tests/smoke-test/test/smoke.test.ts

```diff
@@ -0,0 +1,16 @@
+const smokeTestRoute =
+  'https://smoke-test-app.cfapps.eu12-001.hana.ondemand.com';
+describe('Smoke Test', () => {
+  it('aicore', async () => {
```

Comment:
```suggestion
  it('gets a 200 response from aicore service endpoint', async () => {
```

Reply:
I changed it, but used different test. Your suggestion actually showed me that I am not testing what I think I am testing ;)